### ISSUE TITLE
Caffe version

### ISSUE THREAD

**Issue by vitoralbiero** [xipeng13/recurrent-face-alignment#1]
Hello, I am trying to run your code, but caffe is giving me errors like: "Message type "caffe.LayerParameter" has no field named "bn_param"."
I am using the latest version from the oficial repository.

So, which version of the caffe are you using?

Thank you.

**Comment by xipeng13** [comment]
It needs caffe that support batch normalization layer, such as segnet https://github.com/alexgkendall/caffe-segnet
We update the codes and pre-trained models. Please check it. Thanks!

**Comment by xipeng13** [comment]
Hi Vitor,

The codes need caffe that support batch normalization layer, such as segnet
https://github.com/alexgkendall/caffe-segnet

We will release our distributed version soon.

Thanks,
Xi
